Hunter/Assets/Scripts/Model/Behaviours/IBehaviour.cs

```diff
@@ -0,0 +1,6 @@
+using System.Numerics;
+
+public interface IBehaviour
+{
+    public Vector2 GetForce();
+}
```

Comment:
Interfaces can't be used with static classes
![image](https://user-images.githubusercontent.com/50269023/142856888-ce479fd5-ed46-4c6c-b806-e0a4a17d0ca2.png)

Reply:
+ some behaviours will have different parametrs.
Wander needs object (or We can just pass 1 or 2 arguments and pass one of them back)
Flee and Steer - theoretically only position of two entities
And probably there will be some interesting stuff with Kill
So they will have different contracts